source/threads/scraper.py

```diff
@@ -171,7 +174,20 @@ def get_download_links(self):
 
     def scrape_automated_releases(self):
         base_fmt = "https://builder.blender.org/download/{}/?format=json&v=1"
-        for branch_type in ("daily", "experimental", "patch"):
+
+        branches = ("daily", "experimental", "patch")
+        branches = [
+            "daily/archive" if get_show_daily_archive_builds() and branch == "daily" else branch for branch in branches
+        ]
+        branches = [
+            "experimental/archive" if get_show_experimental_archive_builds() and branch == "experimental" else branch
+            for branch in branches
+        ]
+        branches = [
+            "patch/archive" if get_show_patch_archive_builds() and branch == "patch" else branch for branch in branches
+        ]
```

Comment:
This could probably be done using a single tuple instead of multiple for loops

Reply:
I knew there was a better way to do it 😅